plugins/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: plugin
+title: Plugins
+---
+# 1. Plugins
+
+Plugins are tools to extend and modify the functionality of Foreman. The core of Foreman is designed to be lean, to maximize flexibility and minimize code bloat. Plugins offer custom functions and features so that each user can tailor their environment to their specific needs. Foreman plugins are implemented as [rail engines](http://guides.rubyonrails.org/engines.html) that are packaged as [gems](http://guides.rubygems.org/what-is-a-gem/) and thus easily installed into Foreman.
```

Comment:
Ahhh ok. Parrallel structure is to just the same pattern of words especially when listing 2 or more items. In other words, the first items start with "to", so that last one should as well (to be lean, to maximize flexibility and to minimize code bloat).